Gemfile

```diff
@@ -72,7 +72,7 @@ gem 'premailer-rails'
 gem 'rack-attack', '~> 6.6'
 gem 'rack-cors', '~> 1.1', require: 'rack/cors'
 gem 'rails-i18n', '~> 6.0'
-gem 'rails-settings-cached', '~> 0.6'
+gem 'rails-settings-cached', '~> 0.6', git: 'https://github.com/zunda/rails-settings-cached.git', branch: 'v0.6.6-aliases-true'
```

Comment:
Is there an issue opened to update to a more recent version of this gem?

This could be a temporary workaround but the latest version (2.x, so 2 major bumps to handle!) would be a much better solution (and will be needed for upgrading Rails as well)

Reply:
There _was_ an issue, and I'll try to find it as a reference.  IIRC the problem was that the new version is very *not* backwards compatible, and the effort to upgrade was too much.

Reply:
I couldn't find an issue, only [this comment](https://github.com/mastodon/mastodon/pull/21061#issuecomment-1341779139) and [this comment](https://github.com/mastodon/mastodon/pull/22585#discussion_r1057907260).

I'll file an issue so that we can track it.

Reply:
I made an issue here: https://github.com/mastodon/mastodon/issues/23599

I filed it as a feature request because it didn't seem to fit the "Bug" template.  I was tempted to put it in discussions, but technical debt didn't really seem to fit in there either.

If you need me to change the location (or close it), please let me know.